examples/simple-platform/structure.md

```diff
@@ -0,0 +1,47 @@
+# Platform structure
+
+Our example has two React applications: `app-account` and 
+`interbit-template`. These apps are served in the user's browser as any other 
+React app would be. Either there is a static bundle served in a production 
+environment, or they are served from their own webpack servers in development 
+mode. Each React app starts an Interbit node locally, in the browser. The 
+platform also runs an Interbit node each for the `platform-deploy` and 
+`web-auth-endpoint` hypervisors. 
+
+The `platform-deploy` node hosts the accounts and template chains. (See 
+[Chain Sponsorship](../../reference/interbit-middleware/chainSponsorship.adoc) 
```

Comment:
Remove the `../..` before `/reference`. GitBook knows where the top-level folder containing the documentation lives, and using an absolute path makes it easier if we need to move this example up or down a level.